README.md

```diff
@@ -713,6 +713,21 @@ fastify.get('/', async (request, replay) => {
 
 ```
 
+## `jsonwebtoken` migration changes
```

Comment:
We should use something like: 'Upgrading from `fastify-jwt@3`'
WDYT ?

Reply:
Yeah that could be also - I have no strong preference. Maybe others can say something as well?
@simoneb  @Eomm

Reply:
yes let's specify which version we're migrating from. note that this could also live in a standalone doc instead of cluttering the main readme

Reply:
Done